examples/with-external-scoped-css/.editorconfig

```diff
@@ -0,0 +1,11 @@
+# editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
```

Comment:
2-space indentation is used in Next.js code-base, so it would be nice to follow the style in the examples as well. Also, I would drop this config to keep the example clean.